cache/cache.go

```diff
@@ -97,6 +100,33 @@ func (r *RedisClient) GetBool(key string) (bool, error) {
 	return strconv.ParseBool(result)
 }
 
+// SetInt saves an integer value to REDIS.
+// If the client was set up with a prefix it will be added in front of the key.
+// Zero expiration means the key has no expiration time.
+func (r *RedisClient) SetInt(key string, value int64, expiration time.Duration) error {
+	return r.Set(key, strconv.FormatInt(value, 10), expiration)
+}
+
+// GetInt retrieves an integer value from REDIS.
+// If the client was set up with a prefix it will be added in front of the key.
+func (r *RedisClient) GetInt(key string) (int64, error) {
+	result, err := r.Get(key)
```

Comment:
🙈🙈🙈🙈
`r.Get` is a function from this package, it is not `r.Redis.Get` so `redis.Nil` was already `ErrNotFound`. Same applies for `GetBool` were you just added the conversion a couple of days ago.